include/translatable-object-with-types-interface.php

```diff
@@ -9,6 +9,13 @@
  * Interface to use for objects that can have one or more types.
  *
  * @since 3.4
+ *
+ * @phpstan-type DBInfo array{
```

Comment:
To avoid mixing ourselves with the  DBInfo of `PLL_Translatable_Object`
```suggestion
 * @phpstan-type DBInfoWithType array{
```